.azure/pipelines/ci.yml

```diff
@@ -9,6 +9,7 @@ trigger:
     include:
     - master
     - release/*
+    - internal/release/3.*
```

Comment:
I thought we were only supposed to update the PR section and not the trigger section

Reply:
Originally that was the case, but @mmitche and I have since come up with a new scheme & changed our minds on this one.